assets/src/edit-story/components/panels/design/sizePosition/sizePosition.js

```diff
@@ -25,29 +25,46 @@ import { __, _x } from '@web-stories-wp/i18n';
 /**
  * Internal dependencies
  */
-import { MULTIPLE_VALUE } from '../../../../constants';
-import { BoxedNumeric, Button, Row, Toggle } from '../../../form';
+import {
+  LockToggle,
+  NumericInput,
+  Tooltip,
+  PLACEMENT,
+  Icons,
+} from '../../../../../design-system';
+import { MULTIPLE_DISPLAY_VALUE, MULTIPLE_VALUE } from '../../../../constants';
 import { dataPixels } from '../../../../units';
-import { Lock as Locked, Unlock as Unlocked } from '../../../../icons';
-import useStory from '../../../../app/story/useStory';
 import { getDefinitionForType } from '../../../../elements';
 import { calcRotatedObjectPositionAndSize } from '../../../../utils/getBoundRect';
 import { SimplePanel } from '../../panel';
 import FlipControls from '../../shared/flipControls';
-import { getMediaBaseColor } from '../../../../utils/getMediaBaseColor';
 import { getCommonValue, useCommonObjectValue } from '../../shared';
 import usePresubmitHandlers from './usePresubmitHandlers';
 import { getMultiSelectionMinMaxXY, isNum } from './utils';
 import { MIN_MAX, DEFAULT_FLIP } from './constants';
 
-const StyledToggle = styled(Toggle)`
-  margin: 0 10px;
+const Grid = styled.div`
+  display: grid;
+  grid-template-areas:
+    'x . . . y . .'
+    'w . d . h . l'
+    'r . . . f . .';
+  grid-template-columns: 112px 1fr 8px 1fr 112px 1fr 32px;
+  grid-template-rows: repeat(3, 36px);
+  row-gap: 16px;
+  align-items: center;
+  justify-items: start;
```

Comment:
I dedided that grid was the best way to layout all of this to also be able to reduce spacing when there's a scrollbar while also keeping both rows and columns properly aligned.